examples/uki/extra/etc/mkinitcpio.conf

```diff
@@ -0,0 +1,3 @@
+MODULES=(overlay erofs)
+BINARIES=(fstrace)
```

Comment:
This definitely looks like a typo between `fsck` and `strace`.

I guess you could remove this entirely, though.  We don't actually need strace in there.  Up to you.

Reply:
Let's leave it in for further breakage :)